pluginClientSdkLib/build.gradle

```diff
@@ -8,7 +8,7 @@ dependencies {
     // plugin so instead the dependency is filtered out from the POM
     // https://code.google.com/p/android/issues/detail?id=68316
     compile group:'com.android.support', name:'support-v4', version:']19,)'
-
+    compile group:'com.android.support', name:'appcompat-v7', version:'[21,)'
```

Comment:
Should the version matcher for support-v4 and appcompat-v7 be the same?

Both are bundled in the local maven repo installed by the Android SDK manager, so it would be unlikely for version 20 of support-v4 to be used with version 21 of app-compat in practice.  But I still think the two libraries probably should be in sync

Also the ] in front of support-v4's version matcher was an artifact of the Lollipop preview release causing some Gradle grief.  '[21,)' should be OK for both support-v4 and appcompat-v7.